### PR TITLE
Support command-line arguments running with Docker

### PR DESCRIPTION
**Pull request by JohnNiang**
#### What type of PR is this?

/kind improvement
/area core

#### What this PR does / why we need it:

Now, we can pass command-line arguments while using Docker. Please refer to <https://github.com/halo-dev/halo/issues/2902>.

#### Which issue(s) this PR fixes:

Fixes https://github.com/halo-dev/halo/issues/2902

#### Special notes for your reviewer:

Steps to test:

1. Run Halo with command-line arguments:

    ```bash
    docker run --rm -it -p6666:6666 johnniang/halo:support-command-line-args --server.port=6666
    ```

1. Request <http://localhost:6666>

    ```bash
    curl -v http://localhost:6666
    ```

#### Does this PR introduce a user-facing change?

```release-note
支持命令行参数启动 Halo
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,4 +20,4 @@ ENV JVM_OPTS="-Xmx256m -Xms256m" \
 RUN ln -sf /usr/share/zoneinfo/$TZ /etc/localtime \
     && echo $TZ > /etc/timezone
 
-ENTRYPOINT java ${JVM_OPTS} -Djava.security.egd=file:/dev/./urandom org.springframework.boot.loader.JarLauncher
\ No newline at end of file
+ENTRYPOINT ["sh", "-c", "java ${JVM_OPTS} org.springframework.boot.loader.JarLauncher ${0} ${@}"]
```
Comment (JohnNiang) [inline Dockerfile:23]: Please refer to <https://spring.io/guides/topicals/spring-boot-docker/>.

**Review by guqing — COMMENTED** [review]
/lgtm

**Review by ruibaby — APPROVED** [review]
/lgtm
/approve

**Pull request merged**